lib/Backends/CPU/CPUDeviceManager.h

```diff
@@ -40,9 +41,15 @@ class CPUDeviceManager : public QueueBackedDeviceManager {
   const uint64_t functionCost_{1};
 
 public:
-  CPUDeviceManager(const DeviceConfig &config)
+  explicit CPUDeviceManager(const DeviceConfig &config)
       : QueueBackedDeviceManager(config),
-        maxMemoryBytes_(config_.getDeviceMemory(2000000000)) {}
+        maxMemoryBytes_(config_.getDeviceMemory(2000000000)) {
+    Stats.incrementCounter("accelerators_used.cpu", 1);
```

Comment:
how do we test this? (counters working properly in places how they supposed to work).

Reply:
I'll extend the `StatsExporterTest`.  It's really easy since the global exporter supports registration, you can just register a mock exporter, create a device manager, and then check the stat value.

Reply:
Can we verify those counters in a corresponding device manager test?